core/assets/bundles/bundle_nl.properties

```diff
@@ -17,28 +17,28 @@ screenshot.invalid = Map is te groot, Mogelijk niet genoeg ruimte op apparaat.
 gameover = Spel afgelopen
 gameover.pvp = het[accent] {0}[] team heeft gewonnen!
 highscore = [accent]Nieuw topscore!
-copied = Copied.
+copied = Gekopieerd.
 load.sound = Geluid
 load.map = Mappen
 load.image = Afbeeldingen
-load.content = inhoud
+load.content = Inhoud
 load.system = Systeem
 load.mod = Mods
 schematic = Schematic
 schematic.add = Save Schematic...
 schematics = Schematics
-schematic.replace = A schematic by that name already exists. Replace it?
+schematic.replace = Een Schematic met deze naam bestaat al. Wil je het vervangen?
```

Comment:
Schematic is not a Dutch word to my knowledge. You could use "Blauwdruk" instead.

Reply:
ohw, yeah, I thought there was something, I just wasn't sure about it. Thnx, I will implement it